src/network/providers/ethereum.js

```diff
@@ -912,6 +934,20 @@ ethereum.resolveDomain = async (domainName, network = 'rinkeby') => {
     return {owner, content};
 };
 
+/**
+ * Retrieve ethereum domain (ENS) owned by `address`.
+ *
+ * An ENS name requries additional configuration to setup a reverse record,
+ * they are not automatically set up.
```

Comment:
ENS reverse record must be manually set up by the domain owner for this to work.
![ens_reverse-record](https://user-images.githubusercontent.com/101118664/181097927-d815af90-90eb-4913-8d8c-9b1e84773da8.png)